models/wholeBody_ct_segmentation/docs/README.md

```diff
@@ -32,13 +32,28 @@ The segmentation of 104 tissues is formulated as voxel-wise multi-label segmenta
 
 The training was performed with the following:
 
-- GPU: 32 GB of GPU memory
+- GPU: 48 GB of GPU memory
 - Actual Model Input: 96 x 96 x 96
 - AMP: True
 - Optimizer: AdamW
 - Learning Rate: 1e-4
 - Loss: DiceCELoss
 
+### GPU consumption warning:
```

Comment:
I would move this section down to "### Latencies and memory performance of using the bundle with MONAI Label:" section with this updated text.

"### GPU Consumption Warning

Please note that the model is trained to predict 104 structures using 104 classes, which can lead to high GPU consumption when running on a single instance.

For optimal performance, refer to the benchmarking results below for the inference pipeline. Keep in mind that a CT scan with 300 slices typically requires around 27GB of memory. If you're working with larger CT scans, make sure to allocate more GPU memory or consider using the CPU for inference."